tests/test_basic_graphs.py

```diff
@@ -0,0 +1,56 @@
+"""
+This test is created to check how a single molecule SMIRKS graph works
+"""
+
+from chemper.graphs.cluster_graph import ClusterGraph
+from chemper.graphs.fragment_graph import ChemPerGraphFromMol, ChemPerGraph
+from chemper.mol_toolkits import mol_toolkit
+import pytest
+
+
+@pytest.mark.parametrize('graph_method', [ClusterGraph, ChemPerGraph])
+def test_empty_graph(graph_method):
+    """
+    Test basic function of empty graphs
+    """
+    c = graph_method()
+    assert c.as_smirks() is None
+
+    new_atom = c.add_atom(None)
+    assert c.as_smirks() == '[*]'
+
+    newer_atom = c.add_atom(None, None, new_atom)
+    assert c.as_smirks() == '[*]~[*]'
+
+    c = graph_method()
+    c.add_atom(None, new_smirks_index=1)
+    assert c.as_smirks() == '[*:1]'
+
+
+# make sure corner cases can at least be built without failing
```

Comment:
Why are these corner cases?

Reply:
Corner cases was maybe not the best word for them. They were mostly added to increase coverage. Now that I think about it the `test_no_fail_fragment` could probably also have expected smirks patterns for at least the 1 layer options, so maybe those should go above. 

The multiple layers for the cluster graphs become really really complicated to write out and because decorators are stored as sets and not lists the SMIRKS patterns aren't consistent, I could maybe make a sorting option that makes the smirks patterns consistent so that more intense testing can be done instead of just "does it not fail"